simbev/car.py

```diff
@@ -933,18 +933,6 @@ def export(self, region_directory, simbev):
         """
 
         for charge_event in self.grid_timeseries_list:
-            if charge_event["time"] == 0:
-                charge_event["park_ts_end"] = (
-                    charge_event["park_ts_end"]
-                    if (charge_event["park_ts_end"] - charge_event["start"])
-                    * simbev.step_size
-                    / 60
-                    <= simbev.occupation_time_max
-                    else int(
-                        charge_event["start"]
-                        + simbev.occupation_time_max * 60 / simbev.step_size
-                    )
-                )
```

Comment:
What did this do and why is it removed?

Reply:
This limits the maximum occupation_time of a charging-event. Now the charging_event it self is limited, so this is not needed anymore.